src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Details.cshtml

```diff
@@ -22,6 +22,23 @@
         </div>
     </div>  
 }
+else
+{
+    if (!Model.EmployerCanApprove && Model.DraftApprenticeshipsCount > 0)
```

Comment:
As per comment on the other PR, a single IsCompleteForEmployer that includes cohorts not being empty will simplify this line